### ISSUE TITLE
Unable to consistently generate orders

### ISSUE THREAD

**Issue by rodrigoprimo** [woocommerce/wc-smooth-generator#5]
I'm having issues when trying to generate orders. If I try to generate a relatively small number of orders, the command sometimes works, but sometimes it doesn't work.

I get two different errors when trying to run `wp wc generate orders`:

```
vagrant@vvv:/vagrant/www/wccore$ wp wc generate orders 1
mmap() failed: [12] Cannot allocate memory
mmap() failed: [12] Cannot allocate memory
(...)
mmap() failed: [12] Cannot allocate memory
vagrant@vvv:/vagrant/www/wccore$
```

Or just a silent fail:

```
vagrant@vvv:/vagrant/www/wccore$ wp wc generate orders 1
vagrant@vvv:/vagrant/www/wccore$ 
```

**Comment by AmandaRiu** [comment]
I'm also getting errors. Usually the requests to create smaller batches of orders (less than 20) will finish, but larger batches will crash with this:
```
user102beda6@jurassic-ninja2:~/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator$ wp wc generate orders 20
Generating orders  95 % [=========================================================================================================================================================>        ] 0:05 / 0:06PHP Fatal error:  Uncaught Error: Cannot instantiate abstract class Faker\Provider\Text in /srv/users/user102beda6/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator/vendor/fzaninotto/faker/src/Faker/Factory.php:22
Stack trace:
#0 /srv/users/user102beda6/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator/includes/Generator/Customer.php(42): Faker\Factory::create(0)
#1 /srv/users/user102beda6/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator/includes/Generator/Order.php(114): WC\SmoothGenerator\Generator\Customer::generate(true)
#2 /srv/users/user102beda6/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator/includes/Generator/Order.php(43): WC\SmoothGenerator\Generator\Order::get_customer()
#3 /srv/users/user102beda6/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator/includes/CLI.php(67): WC\SmoothGenerator\Generator\Order::generate()
#4 [internal function]: WC\SmoothGenerator\CLI->orders(Array, Array)
#5 phar:///opt/sp/bin/wp-cli.phar/php/WP_CLI/Dispatcher/Command in /srv/users/user102beda6/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator/vendor/fzaninotto/faker/src/Faker/Factory.php on line 22
Fatal error: Uncaught Error: Cannot instantiate abstract class Faker\Provider\Text in /srv/users/user102beda6/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator/vendor/fzaninotto/faker/src/Faker/Factory.php:22
Stack trace:
#0 /srv/users/user102beda6/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator/includes/Generator/Customer.php(42): Faker\Factory::create(0)
#1 /srv/users/user102beda6/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator/includes/Generator/Order.php(114): WC\SmoothGenerator\Generator\Customer::generate(true)
#2 /srv/users/user102beda6/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator/includes/Generator/Order.php(43): WC\SmoothGenerator\Generator\Order::get_customer()
#3 /srv/users/user102beda6/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator/includes/CLI.php(67): WC\SmoothGenerator\Generator\Order::generate()
#4 [internal function]: WC\SmoothGenerator\CLI->orders(Array, Array)
#5 phar:///opt/sp/bin/wp-cli.phar/php/WP_CLI/Dispatcher/Command in /srv/users/user102beda6/apps/user102beda6/public/wp-content/plugins/wc-smooth-generator/vendor/fzaninotto/faker/src/Faker/Factory.php on line 22
```

**Comment by rodrigoprimo** [comment]
Fixed by e9d4a0cf9929903a4f815f31d5c6329d391b7915
